src/data/scripts/pile/convert_the_pile.py

```diff
@@ -0,0 +1,48 @@
+from datasets import Dataset
+from argparse import ArgumentParser
+from typing import Dict, List, Any
+
+import os
+import tqdm
+import json
+
+DATASET_NAME = "pile"
+NULL = "null"
+
+if __name__ == "__main__":
+    parser = ArgumentParser()
+    parser.add_argument("train_jsonl_uncompressed_directory")
```

Comment:
I just recently learn we can open `*.jsonl.zstd` without the need of uncompressing file.

Here is an example to do that
https://github.com/jzhang38/TinyLlama/blob/main/scripts/prepare_slimpajama.py

Reply:
done